xml/System.Net.Http/HttpClientHandler.xml

```diff
@@ -656,8 +656,8 @@ After NuGet package v4.3.2, the default value of <xref:System.Net.DecompressionM
         <ReturnType>System.Security.Authentication.SslProtocols</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets or sets the SSL protocol used by the HttpClient objects managed by the HttpClientHandler object.</summary>
```

Comment:
"SSL protocol" This should be "TLS/SSL protocol".  The current standard is that we really call this TLS now instead of SSL even though the property is named "SslProtocols.